### PR TITLE
feat(*): add k8s version to version debug cmd

### LINKED ISSUE #1398: Add helm debug command
In order to help service users issues better we should create a script that they can use in order to quickly provide maintainers with the info they need in order to debug their problem. It can start simple but could eventually become part of helm itself.

### PR DESCRIPTION
**Pull request by rbwsam**
When 'helm version --debug' is run, this will print the kubernetes server
version as well as the helm client and server versions.

Closes #1398 

---

I'm wide open to feedback - I'm here to learn!

I think it would make sense to also move the tiller version code to the new `InfoService` but I didn't want to include that in this PR since it's probably a breaking change.

### CODE REVIEW

File: pkg/proto/hapi/chart/chart.pb.go [hunk 1]
```diff
@@ -1,5 +1,6 @@
-// Code generated by protoc-gen-go. DO NOT EDIT.
```
Comment (rbwsam) [inline pkg/proto/hapi/chart/chart.pb.go]: I made sure I built this with libprotoc 3.2.0 but the comments are still different...

**Comment by technosophos** [comment]
Sorry for being late on this one. Would it be easier to just run the query from the client directly to the Kubernetes API server instead of adding a Tiller roundtrip? (For example, see the tunnel code). Then we wouldn't have to modify the proto at all.

**Comment by rbwsam** [comment]
Great point! I will make that change and update the PR.

File: cmd/helm/version.go [hunk 2]
```diff
@@ -89,6 +90,13 @@ func newVersionCmd(c helm.Interface, out io.Writer) *cobra.Command {
 }
 
 func (v *versionCmd) run() error {
+	if settings.Debug {
```
Comment (bacongobbler) [inline cmd/helm/version.go]: hey! Sorry about the long wait on the review here.

Would you mind moving this after line 106? If we're making a server-side call to check the kubernetes version, we will want to respect the `--client-only` flag.

**Review by bacongobbler — CHANGES_REQUESTED** [review]

**Comment by jascott1** [comment]
Hi @rbwsam Thanks for the PR! It looks like we are one small change away from being mergeable. Do you think you will be able to continue working on this?

**Comment by rbwsam** [comment]
@jascott1 Yes I am happy to make the changes, expect them in the next 48 hours.

**Review by bacongobbler — APPROVED** [review]
This is a fantastic enhancement. Thanks @rbwsam!

**Pull request merged**